packages/nimble-components/src/table/components/row/index.ts

```diff
@@ -170,6 +170,11 @@ export class TableRow<
         return null;
     }
 
+    public override disconnectedCallback(): void {
+        super.disconnectedCallback();
+        this.removeColumnObservers();
```

Comment:
From a quick (incomplete) search I found at least one other spot we are failing to clean-up notifiers in the disconnected callback: https://github.com/ni/nimble/blob/c4470de36c67bd8b08155fe4dedb1fce96972500/packages/nimble-components/src/table-column/enum-base/index.ts#L127

We should create a tech debt issue to evaluate our `getNotifier` & `subscribe` usages to make sure they clean-up correctly and address in a separate PR.

Reply:
Created #2393